index.html

```diff
@@ -255,6 +255,19 @@ <h3>legend</h3>
           }
         }
 
+        function open_row(row, tr) {
+          var extra_data = format(row.data());
+          if (extra_data !== "") {
+            row.child( format(row.data()) ).show();
```

Comment:
```suggestion
          var extra_data = format(row.data());
          if (extra_data !== "") {
            row.child( extra_data ).show();
```

No point calling the same function twice, right?